jekyll/_cci2/insights-glossary.md

```diff
@@ -0,0 +1,120 @@
+| Total Credits | The total credits consumed across all selected projects in an organization. |
+| Overall Success Rate | The percentage of runs that completed successfully across all runs in the selected projects and workflows. (calculated by `100 x (All Successful Runs/ All Runs)`) |
+{: class="table table-striped"}
+
+## Workflow Metrics
+
+| Term                    | Definition     |
+|-------------------------|----------------|
+| Time to Recovery | Total time spent in a failed state before a successful execution. |
+| Mean Time to Recovery (MTTR)| The average time it takes to get back to a successful execution from a failed one (Calculated by `total recovery time / # of failures`). If there are no failed builds (and thus no recoveries) in the current time window, the MTTR value will be empty. Empty MTTR values will be treated as 0 when calculating Trends. |
+| Throughput | The average (mean) number of builds of any status per day. Calculated by `Sum of builds/ # of days.` |
+{: class="table table-striped"}
+
+
+## Trends
+
+Raw metric values are often difficult to interpret on their own. Trends provide
+additional context to Insights metrics by presenting a relative benchmark
+against previous performance. For instance, on the last 7-day view, trends will
+display the change in value or delta compared to the prior 7-day window.
+
+Please note that trends are available only for 24-hour, 7-day, and 30-day time
+windows. For percentile metrics like `duration`, approximation methods are used
+to find the prior window benchmarks.
+
+Trends are calculated as `100 * (current value - previous value) / prior-value`.
+
+### Duration 
+{:.no_toc}
+
+**P95 Duration** 
+
+For P95 duration, trends data is calculated by using the worst-case performance
+of the previous time window as a benchmark.
+
+For example, to calculate the last 7-day trend information, Insights selects the
+worst/largest daily P95 duration of the previous 7 days as a benchmark. For the
+last 24-hour trends, Insights selects the worst hourly P95 of the previous 24
+hours as a benchmark.
+
+**P50 Duration** 
+
+For P50 duration, Trends compares current duration with the median performance
+of the prior time window. For example, the last 30-day trends for P50 duration
+are calculated by taking the median of the daily P50 values as a benchmark.
+
+
+### Representations of trend data
+
+
+This section describes how your trend data may appear across various metrics.
+
+![]({{ site.baseurl }}/assets/img/docs/insights_trend_data.png)
+<sub>Screenshot depicting how trend data might appear.</sub>
+
+- **Green**: The metric is trending in the right direction.
+- **Red**: The metric is trending in the wrong direction.
+- **Grey**: A trend in either direction is not right or wrong. 
+
+Red and Green are used when describing the `Success Rate`, `Throughput` and `MTTR` metrics. Grey arrows are used when describing `Runs`, `Duration` and `Total Credits`
+
+**Percentages**
```

Comment:
Can we add images to show how users can expect percentages, multiples, arrows, etc to look?